### ISSUE TITLE
Add a "Disable Items" options

### ISSUE THREAD

**Issue by Sewef** [TeamPneumatic/pnc-repressurized#206]
Hello there
I'm working on a modpack and I want to disable the Spawner Agitator.
Remove the recipe with CTweaker is easy, but remove from Dungeon Chests is much harder.
Is it possible to add a Disable config? ^_^

**Comment by desht** [comment]
Yeah, that can be added.  A configurable spawn chance is probably the best bet here.

**Comment by desht** [comment]
Note you can just set "B:enableDungeonLoot" to false now, although that will also disable the Stop! Worm and Nuke Virus, which you may or may not want...

**Comment by Sewef** [comment]
> Note you can just set "B:enableDungeonLoot" to false now, although that will also disable the Stop! Worm and Nuke Virus, which you may or may not want...

Ooooooh, I feel like Marco Polo discovering America o___o It's perfect!

I never used Ressource packs, I'll take a look. Thanks :D

**Comment by desht** [comment]
I'm not a resource pack expert either, but I think it should be possible.  Let me know in any case, since it would be nice to not need to modify code here.

**Comment by Sewef** [comment]
Well, adding the disableItem option in the config file will make modpack making easier, instead of making a ressourcepack for client and server. + changing the config on the go would remove the item from existing worlds, seems better imo.
When I tried to ban some items from dungeonLoot, I did a .function file perma-checking inventories. That was so lame.
